src/main/deltachat/webxdc.ts

```diff
@@ -111,7 +110,7 @@ export default class DCWebxdc extends SplitOut {
 
         if (!accounts_sessions.includes(accountId)) {
           accounts_sessions.push(accountId)
-          ses.protocol.handle('webxdc', async request => {
+          ses.protocol.handle('webxdc', async (request: any) => {
```

Comment:
sure that marking it as any is correct here? shouldn't it be typed automatically by electron types?